cmd/validateRadixConfig.go

```diff
@@ -87,6 +168,7 @@ func init() {
 	validateCmd.AddCommand(validateRadixConfigCmd)
 	validateRadixConfigCmd.Flags().StringP(flagnames.ConfigFile, "f", "radixconfig.yaml", "Name of the radixconfig file. Defaults to radixconfig.yaml in current directory")
 	validateRadixConfigCmd.Flags().BoolP(flagnames.Print, "p", false, "Print parsed config file")
+	validateRadixConfigCmd.Flags().StringP(flagnames.Schema, "s", "https://raw.githubusercontent.com/equinor/radix-operator/release/json-schema/radixapplication.json", "Validate against schema")
```

Comment:
Do we need a shorthand for schema? Most users will never use it, and the more shorthands we add, the more chance of a conflict with other flags later.
Also, add some info about what values this flag accepts: url (https://) or file (file://)?

Reply:
enig! la inn `Validate against schema. http://, file:// or path is supported`, og installerte samme versjon som Radix-API, og på samme måte :)